src/Dhall/Core.hs

```diff
@@ -609,10 +609,35 @@ prettyNatural = Pretty.pretty
 prettyDouble :: Double -> Doc ann
 prettyDouble = Pretty.pretty
 
+-- TODO: Correctly escape multi-line literals
```

Comment:
What's the todo here about? What case isn't dealt with?

Reply:
Oh, whoops!  The TODO is a very real issue I still need to fix